### ISSUE TITLE
Some help needed

### ISSUE THREAD

**Issue by Only4Gamers** [playgameservices/play-games-plugin-for-unity#2989]
Hello everyone,

Recently I added Google Play Games Leaderboard in my game using Play Games Plugin Unity. And I am little confused in some configurations.

### What Scopes do I needed to Add in OAuth Consent Screen?
What Scopes I needed to Add in OAuth Consent Screen Page (Update selected scopes)?
![104410327-1b7fa800-558e-11eb-9116-fd81ec765b32](https://user-images.githubusercontent.com/71321023/104560742-548e4a00-566c-11eb-9b18-457779d71193.PNG)

So I checked 3 scopes:
.../auth/userinfo.email
.../auth/games_lite
openid

Did I choose right scopes for showing Leaderboard? Do I need to choose these scopes at all?

### What is this new Joke?
When I am clicking on PUBLISH APP button in Google Cloud Platform it is showing me this message:
![Capture2](https://user-images.githubusercontent.com/71321023/104560781-666fed00-566c-11eb-9c38-622faa7a8bc3.PNG)

I didn't clicked on Confirm yet. Will I need to Verify my app? While I didn't asked for any sensitive scope.
Please help someone. I previously published my other game 6 months ago and that time I didn't face any of these Jokes.

**Comment by junpeikun** [comment]
Yep, I'm in the same boat and I can't fathom as to why the process got this hard. Personally, I just use achievements and the save game feature. The official tutorial just throws a blurry picture and and asks you to fill the rest. How?

**Comment by Only4Gamers** [comment]
> Yep, I'm in the same boat and I can't fathom as to why the process got this hard. Personally, I just use achievements and the save game feature. The official tutorial just throws a blurry picture and and asks you to fill the rest. How?

Yes, as a game developer now I am used to of this behavior everywhere. And specially when it comes to a product from Google then ready for this attitude for sure. After starting Verification, I receive email from Google that my game don't require Verification and they are showing the above message for no reason. And in scopes I unchecked all scopes and everything is working as expected so far.

**Comment by junpeikun** [comment]
I also received an email today stating that if I don't need an icon for my app, I can skip verification. I replied by the affirmative. Hopefully this speeds up things. So if I understand correctly, now your app is stated as "published" in the consent screen page?

**Comment by Only4Gamers** [comment]
> I also received an email today stating that if I don't need an icon for my app, I can skip verification. I replied by the affirmative. Hopefully this speeds up things. So if I understand correctly, now your app is stated as "published" in the consent screen page?

Yes, it's showing Publishing status, in production.

**Comment by vaibhavgpt-google** [comment]
Closing this issue as part of a cleanup of older tickets. Google Play Games v1 is now deprecated. Please upgrade to version v2.1.0 or newer: https://github.com/playgameservices/play-games-plugin-for-unity/releases/tag/v2.1.0. If this issue persists on the new v2 architecture, please open a new ticket.
